.github/workflows/build-docker.yml

```diff
@@ -130,6 +133,5 @@ jobs:
         working-directory: /tmp/digests
         run: |
           docker buildx imagetools create \
-            -t ${{ env.UV_BASE_IMG }}:latest \
-            -t ${{ env.UV_BASE_IMG }}:${{ (inputs.plan != '' && fromJson(inputs.plan).announcement_tag) || 'dry-run' }} \
+            $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
```

Comment:
Can you add a comment what this does?

Reply:
👍 a comment would be nice here

Reply:
Makes sense, added in 4ed2d6ff5893bf895b1a1c1b432d60e7c9a1a790